ocs_ci/utility/memory.py

```diff
@@ -73,10 +73,13 @@ def _rec_memory(proc: Process):
                 ),
             ]
         )
+    # ZombieProcess's, NoSuchProcess's come too often within a test run,
+    # we're polling each process once per 3 sec. ZombieProcess and NoSuchProcess
+    # appear due to concurrency. Failed polls are not valuable information
     except ZombieProcess:
-        log.exception("Cannot access ZombieProcess. Recording skipped")
+        pass
     except NoSuchProcess:
-        log.exception("Process is unavailable. Recording skipped")
+        pass
```

Comment:
I would propose to move this fix to separated PR as it is not related to the test `test_ceph_pg_log_dups_trim` and also we might want to backport it to the release branches (if it is a fix for Issue https://github.com/red-hat-storage/ocs-ci/issues/7171).

Reply:
reverted the change

Reply:
@dahorak extracted to https://github.com/red-hat-storage/ocs-ci/pull/7181